src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -366,6 +369,24 @@ public enum FileIOPermissionAccess
         Read = 1,
         Write = 2,
     }
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Assembly, AllowMultiple = true, Inherited = false)]
+    public sealed class FileIOPermissionAttribute : CodeAccessSecurityAttribute
+    {
+        public FileIOPermissionAttribute(SecurityAction action) : base(action) { }
+        public string Read { get; set; }
+        public string Write { get; set; }
+        public string Append { get; set; }
+        public string PathDiscovery { get; set; }
+        public string ViewAccessControl { get; set; }
+        public string ChangeAccessControl { get; set; }
+        [Obsolete]
+        public string All { get; set; }
+        // Read, Write, Append, PathDiscovery, but no ACL-related permissions
```

Comment:
oh it's from desktop. can just remove.

---
In reply to: [108824060](https://github.com/dotnet/corefx/pull/17677#discussion_r108824060) [](ancestors = 108824060)